### PR TITLE
Added distance calculating api 

### PR DESCRIPTION
**Pull request by YadavGulshan**
check `api/distance/` endpoint.

### CODE REVIEW

File: backend/pharmacy/api/views/mapper/distance.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                "lat2": "latitude2",
+                "lon2": "longitude2",
+            }
+        })
+
+    def post(self, request):
+        """
+        This method will calculate the distance between two points
+        """
+        lat1 = request.data.get('lat1')
```
Comment (rahul007-bit) [inline backend/pharmacy/api/views/mapper/distance.py]: ye data kese pass hoga body me ki parameter me

File: backend/pharmacy/api/views/mapper/distance.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            d = R.c
+        """
+
+        c = 2*atan2(sqrt(a), sqrt(1-a))
+        d = radius*c
+        """
+        d is the distance between two points
+        """
+
+        return JsonResponse({
```
Comment (rahul007-bit) [inline backend/pharmacy/api/views/mapper/distance.py]: great work 👏🏼👏🏼👏🏼👏🏼👏🏼👏🏼

**Review by rahul007-bit — APPROVED** [review]
Nice work bro, I am proud👏🏼of you!!!!

File: backend/pharmacy/api/views/mapper/distance.py (hunk 1 above)
Comment (YadavGulshan) [reply backend/pharmacy/api/views/mapper/distance.py]: Body

**Pull request merged**